Tasks/Liaskovski/Task1/task1.py

```diff
@@ -0,0 +1,6 @@
+#Task1
+My_deposit = 20000
+Rate = 15
+Term = 5 
```

Comment:
these values should be implemented as decimal since they are related to money (in order to prevent errors during calculation), -1 point for the unwise choice